src/core/Internal.ml

```diff
@@ -131,33 +131,32 @@ module Make(Plugin : PLUGIN)
     let[@inline] assigned l = l.assigned
     let[@inline] weight l = l.l_weight
 
-    let make (st:st) (t:term) : t =
+    let make (st : st) (t : term) : t =
       try MT.find st.t_map t
       with Not_found ->
-        let res = {
-          lid = st.cpt_mk_var;
-          term = t;
-          l_weight = 1.;
-          l_idx= -1;
-          l_level = -1;
-          assigned = None;
-        } in
+        let res =
+          { lid = st.cpt_mk_var;
+            term = t;
+            l_weight = 1.;
+            l_idx = -1;
+            l_level = -1;
+            assigned = None }
```

Comment:
I quite like the original style of the code for creating multi-lines records, is there an ocamlformat option that would keep it ?

Reply:
I didn't find anything for that, and I agree that the old style is really nice :frowning_face: 
maybe we should ask ocamlformat's maintainers their opinion on that?

Reply:
Agreed, I'll do that after my holidays if you don't have the time/motivation

Reply:
So, while reading the discuss forum, I found out that what we want for records such as that, it the `after-and-docked` value for the `break-separators` option.